src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -23,46 +19,43 @@ import usePermissions from '@hooks/usePermissions';
 import useScreenWrapperTranstionStatus from '@hooks/useScreenWrapperTransitionStatus';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import type {MaybePhraseKey} from '@libs/Localize';
+import type {Options} from '@libs/OptionsListUtils';
 import * as OptionsListUtils from '@libs/OptionsListUtils';
 import * as Policy from '@userActions/Policy';
 import * as Report from '@userActions/Report';
+import type {IOUAction, IOURequestType, IOUType} from '@src/CONST';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {Participant} from '@src/types/onyx/IOU';
 
-const propTypes = {
+type MoneyRequestParticipantsSelectorProps = {
     /** Callback to request parent modal to go to next step, which should be split */
-    onFinish: PropTypes.func.isRequired,
+    onFinish: (value?: string) => void;
 
     /** Callback to add participants in MoneyRequestModal */
-    onParticipantsAdded: PropTypes.func.isRequired,
-
+    onParticipantsAdded: (value: Participant[]) => void;
     /** Selected participants from MoneyRequestModal with login */
-    participants: PropTypes.arrayOf(
-        PropTypes.shape({
-            accountID: PropTypes.number,
-            login: PropTypes.string,
-            isPolicyExpenseChat: PropTypes.bool,
-            isOwnPolicyExpenseChat: PropTypes.bool,
-            selected: PropTypes.bool,
-        }),
-    ),
+    participants?: Participant[];
 
     /** The type of IOU report, i.e. split, request, send, track */
-    iouType: PropTypes.oneOf(lodashValues(CONST.IOU.TYPE)).isRequired,
+    iouType: IOUType;
 
     /** The expense type, ie. manual, scan, distance */
-    iouRequestType: PropTypes.oneOf(lodashValues(CONST.IOU.REQUEST_TYPE)).isRequired,
+    iouRequestType: IOURequestType;
 
     /** The action of the IOU, i.e. create, split, move */
-    action: PropTypes.oneOf(lodashValues(CONST.IOU.ACTION)),
-};
-
-const defaultProps = {
-    participants: [],
-    action: CONST.IOU.ACTION.CREATE,
+    action?: IOUAction;
```

Comment:
Looking at the code this can be a required prop

Reply:
updated to required